src/test/java/com/wikia/webdriver/elements/mercury/components/Search.java

```diff
@@ -2,55 +2,42 @@
 
 import com.wikia.webdriver.common.core.Assertion;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.wikia.webdriver.common.core.Assertion'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)